### PR TITLE
Files workflow

### PR DESCRIPTION
**Pull request by kpsherva**
* closes https://github.com/CERNDocumentServer/cds-rdm/issues/175

### CODE REVIEW

File: cds_migrator_kit/migration_config.py [hunk 1]
```diff
@@ -355,3 +355,4 @@ def _(x):  # needed to avoid start time failure with lazy strings
 base_path = os.path.dirname(os.path.realpath(__file__))
 logs_dir = os.path.join(base_path, "tmp/logs/")
 CDS_MIGRATOR_KIT_LOGS_PATH = logs_dir
+INVENIO_CDS_MIGRATOR_KIT_STREAM_CONFIG = "cds_migrator_kit/rdm/migration/streams.yaml"
```
Comment (jrcastro2) [inline cds_migrator_kit/migration_config.py:358]: minor: I would remove invenio
```suggestion
CDS_MIGRATOR_KIT_STREAM_CONFIG = "cds_migrator_kit/rdm/migration/streams.yaml"
```

File: cds_migrator_kit/rdm/migration/load.py [hunk 2]
```diff
@@ -7,11 +7,19 @@
 
 """CDS-RDM migration load module."""
 
+import os
```
Comment (jrcastro2) [inline cds_migrator_kit/rdm/migration/load.py:10]: Minor: This can be removed

**Review by jrcastro2 — APPROVED** [review]
LGTM! 🚀 

Just a couple of minor comments

**Review by zzacharo — APPROVED** [review]

File: scripts/copy_collection_files.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                    full_path = legacy_record_file["full_path"]
+                    # important: last slash
+                    path_to_replace = "/opt/cdsweb/var/data/files/"
+
+                    rel_path = full_path.replace(path_to_replace, "")
+                    destination_path = os.path.join(destination_prefix, rel_path)
+                    parent_dest_path = os.path.dirname(destination_path)
+                    if not os.path.exists(parent_dest_path):
+                        os.makedirs(parent_dest_path)
+                    shutil.copy(full_path, destination_path)
```
Comment (zzacharo) [inline scripts/copy_collection_files.py:25]: discussed to use the eosclient and permission based tokens that should replace the copy command here and the need for `kinit`

File: cds_migrator_kit/rdm/migration/README.md [hunk 4]
```diff
@@ -135,8 +135,32 @@ current_rdm_records_service.indexer.bulk_index((rec.id for rec in records))
...
+
+
+
+### Full migration workflow of one collection
+
+#### Legacy
+
+```shell
+ssh cds-wn-31 # inveniomigrator tool installed here
+kinit cdsrdmeosdev
```
Comment (zzacharo) [inline cds_migrator_kit/rdm/migration/README.md:153]: see comment about using eosclient instead

**Pull request merged**